client/src/components/Profile/Profile.js

```diff
@@ -1,8 +1,49 @@
-import React from "react";
+import { React, useEffect, useState } from "react";
 import PropTypes from "prop-types";
+import { Typography } from "@material-ui/core";
+import * as fetcher from "../../lib/fetch";
 
 function Profile({ id }) {
-  return <>Profile of user with id {id}</>;
+  const [loaded, setLoaded] = useState(false);
+  const [userExists, setUserExists] = useState(true);
+  const [name, setName] = useState("");
```

Comment:
Now that we have `loaded`, you could probably combine these two into the same state `user`. If `loaded && !user`, it signifies that the user does not exist.